### ISSUE TITLE
option to send message with strg+enter

### ISSUE THREAD

**Issue by LuckyTurtleDev** [Nheko-Reborn/nheko#1445]
### The Problem

I have seen that their exist an option to send message with shift+enter, but Element and Github does use strg+enter for this.
This different behavior is very confusing. It would be nice to have a more consistent user interface at the matrix-eco-system.

### The Solution

I would recommend to also add an option to use strg+enter.

### Happens in the latest version

- [X] Yes, this feature is still missing.

**Comment by LorenDB** [comment]
I agree that Ctrl+Enter would make more sense; however, there are probably people who will complain if we change this because it will break their muscle memory :upside_down_face:

**Comment by LuckyTurtleDev** [comment]
Best solution would be to let the user choose between strg and shift.

**Comment by xaea-twlv** [comment]
Hey I am a newbie in the open-source space and would like to start contributing to this project it would be great if you could guide me on how I can access this issue in the repo, thank you

**Comment by LorenDB** [comment]
You probably should look at https://github.com/Nheko-Reborn/nheko/blob/master/resources/qml/MessageInput.qml#L169-L190 to get started. Also check out src/UserSettingsPage.h and src/UserSettingsPage.cpp, because you will probably want to add a new setting to toggle this.

With that being said, I have a PR in place to add editable shortcuts, and that will likely be the best way to fix this issue, so I'll just throw out a warning that this might not be a good issue if you want something that will be useful long-term :)
